### DISCUSSION TITLE
Changelog of packages

### DISCUSSION THREAD

**Discussion by joaopedrodcf** [radix-ui/primitives#2479] [Help] [answered]
Hey, first thanks for this awesome and powerful library! 🥇 

I tried to find in the repository a changelog but couldn't find it, wanted for example to know what changed from the select from v1 to v2 to see if there were some breaking changes that I need to be aware of.

As I didn't find it, I then checked the changes in select component and noticed this: 
<img width="1541" alt="image" src="https://github.com/radix-ui/primitives/assets/11476152/0171a526-a5ed-4977-aac2-f69d2da486fe">
Where is the reason why this is a major.

Is there any changelog? Or plans of having one?

Thanks, in advance.

**Comment by benoitgrelard** [comment, marked as answer]
The changelog lives here: https://www.radix-ui.com/primitives/docs/overview/releases
I believe this specific change hasn't been publicly released yet.

**Reply by joaopedrodcf** [reply]
Thanks for the reply ✌️

**Comment by aaron-wilder** [comment]
Yeah, but it's behind. As an example NPM lists the current version of `@radix-ui/react-dialog` as 1.1.4 but, the releases page only goes up to 1.1.2 🤔

**Reply by diced-dev** [reply]
Yep this is the same issue I'm seeing as well. I'm trying to upgrade to React 19 but there is no clear indicator of what will break if I update the radix dependencies
